libs/ledgerjs/packages/cryptoassets/src/data/eip712.ts

```diff
@@ -0,0 +1 @@
+export default {"1:0x7f268357a8c2552623316e2562d90e642bb538e5:d8e4f2bd77f7562e99ea5df4adb127291a2bfbc225ae55450038f27f":{"contractName":{"label":"OpenSea Listing","signature":"304402205f01e62b2331ecb9f1cb3f49a764d66015937cb20cb4c3af625146905ca4832402205076eeb4a74619b4552278836b548c8e525813faa9008399e43bfed7db016408"},"fields":[{"label":"Contract address","path":"exchange","signature":"3044022069df3b1c82dda9849a3df4cd0dd263d2b6c35e4fb8946865a35f5555830971b802201880d72b3d956867b597a2f5097d7f8799bad6e636d3de4030515634bba47155"},{"label":"Price","path":"basePrice","signature":"3045022100db49dbd8426193b6012781f43c3b445d1cf845d99d738091c1f068951728333402204326a2da5203f25deec049deba760bb31a86507c73fe6828c68e0db2eabfc966"},{"label":"Offer expiration","path":"expirationTime","signature":"3045022100e71a4921740fcdf4e489c517fc82f9b7e83cfbd675dff318acb91fbf8eaf175402203fac43650771c6523dad8d995ef414d035a92d262a9cf95885ef2704337a2382"}]}};
```

Comment:
@lambertkevin Just a future improvement opportunity, this could now be typed, to be consistent with other CAL data.